src/service-catalog-appregistry-stack.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { Application } from '@aws-cdk/aws-servicecatalogappregistry-alpha';
+import { Aws, CfnCondition, CfnOutput, CfnParameter, CfnResource, Fn, Stack, Tags } from 'aws-cdk-lib';
+import { StackProps } from 'aws-cdk-lib/core/lib/stack';
+import { Construct } from 'constructs';
+import { OUTPUT_SERVICE_CATALOG_APPREGISTRY_APPLICATION_ARN } from './common/constant';
+import { SolutionInfo } from './common/solution-info';
+
+export interface ServiceCatalogAppRegistryProps extends StackProps {
+}
+
+export class ServiceCatalogAppregistryStack extends Stack {
+  constructor(scope: Construct, id: string, props?: ServiceCatalogAppRegistryProps) {
+    super(scope, id, props);
+
+    const featureName = 'AppRegistry';
```

Comment:
The spoke id(`appreg`) is limited to 3 characters.

Reply:
change to `reg`